src/css/components/_captions-settings.scss

```diff
@@ -35,7 +77,7 @@
 
 .vjs-text-track-settings legend {
   color: $primary-foreground-color;
-  margin: 0 0 5px 0;
+  margin: 0 0 5px;
```

Comment:
can you revert this change and the ones below it? It'll keep the PR focused on just necessary code changes.

Reply:
Yup my code beautifier messed up a few things like these. Working right now to get it back to how it should be